### PR TITLE
Add wifi_blinky.rs for easy beginners start.

### PR DESCRIPTION
**Pull request by henrikberg**
I got a RP Pico W and wanted to verify it's function. The only easy output is a LED, but it has moved from the standard PR Pico. I also spent some time to discover how the button was connected, only to find that button.rs use a normal GPIO.

- Add rp/src/bin/wifi_blinky.rs for easy beginners start.
- Document external button.in rp/src/bin/button.rs.

### CODE REVIEW

**Review by Dirbaio — APPROVED** [review]
Thanks!

Can you rebase on top of main? that contains the fix for the CI failure.

File: examples/rp/src/bin/button.rs [hunk 1]
```diff
@@ -9,9 +9,12 @@ use {defmt_rtt as _, panic_probe as _};
 #[embassy_executor::main]
 async fn main(_spawner: Spawner) {
     let p = embassy_rp::init(Default::default());
-    let button = Input::new(p.PIN_28, Pull::Up);
     let mut led = Output::new(p.PIN_25, Level::Low);
 
+    // Use PIN_28, Pin34 on J0 for RP Pico, as a input.
+    // You need to ad your own button.
```
Comment (Dirbaio) [inline examples/rp/src/bin/button.rs:15]: add

Comment (Dirbaio) [inline examples/rp/src/bin/button.rs:15]:
```suggestion
    // You need to add your own button.
```

**Comment by henrikberg** [comment]
Why is ci/build failing on:
```
error[E0425]: cannot find function `set_max_level` in crate `log`
    --> /ci/code/examples/rp/src/bin/usb_logger.rs:18:5

``` 
and what do I do about it? Do I need to compile all --bin in examples/rp to make sure they pass?

**Comment by henrikberg** [comment]
I'm feeling so proud! All green checks. :-)

**Pull request merged**